server/service/microsoft_mdm.go

```diff
@@ -571,6 +608,30 @@ func (svc *Service) GetMDMWindowsPolicyResponse(ctx context.Context, authToken s
 	return &policyMsg, nil
 }
 
+// GetMDMWindowsEnrollResponse returns a valid RequestSecurityTokenResponseCollection message
+func (svc *Service) GetMDMWindowsEnrollResponse(ctx context.Context, authToken string) (*fleet.RequestSecurityTokenResponseCollection, error) {
+	if len(authToken) == 0 {
+		return nil, fleet.NewInvalidArgumentError("enroll response", "authToken is empty")
+	}
+
+	// Validate the binary security token
+	err := validateBinarySecurityToken(ctx, authToken, svc)
+	if err != nil {
+		return nil, ctxerr.Wrap(ctx, err, "validate binary security token")
+	}
+
+	// Token is authorized
+	svc.authz.SkipAuthorization(ctx)
+
+	// Getting the RequestSecurityTokenResponseCollection message content
+	enrollMsg, err := NewRequestSecurityTokenResponseCollection("test")
```

Comment:
This is likely temporary as this is still in draft, but just in case, the token will need to be changed to the proper one.

Reply:
Yes, this was a temporary placeholder. It was updated on the latest commits